package.json

```diff
@@ -71,6 +72,9 @@
     "webpack-cli": "3.3.10",
     "zen-observable": "0.8.15"
   },
+  "dependencies": {
+    "find-packages": "7.0.2"
```

Comment:
that's a devDependency because everything will be bundled into on file by using webpack.

Reply:
ah okay didn't know that as it's not that common for server modules, I'll move that dependency asap., thanks!